include/functions.h

```diff
@@ -1689,7 +1696,7 @@ s16 ActorCutscene_GetAdditionalCutscene(s16 index);
 s16 ActorCutscene_GetLength(s16 index);
 // void func_800F2138(void);
 // void func_800F2178(void);
-// s16 ActorCutscene_GetCurrentCamera(void);
+s16 ActorCutscene_GetCurrentCamera(s16);
```

Comment:
```suggestion
s16 ActorCutscene_GetCurrentCamera(s16 arg0);
```

Reply:
```suggestion
s16 ActorCutscene_GetCurrentCamera(s16 index);
```
Actually, that should be fine